Jenkinsfile

```diff
@@ -322,27 +322,21 @@ pipeline {
                 }
             }
         }
-        stage('Get Commit Message') {
-            steps {
-                script {
-                    env.COMMIT_MESSAGE = sh(script: 'git show -s --format=%B',
-                                            returnStdout: true).trim()
-                    Map pragmas = [:]
-                    // can't use eachLine() here: https://issues.jenkins.io/browse/JENKINS-46988/
-                    env.COMMIT_MESSAGE.split('\n').each { line ->
-                        String key, value
-                        try {
-                            (key, value) = line.split(':', 2)
-                            if (key.contains(' ')) {
-                                return
-                            }
-                            pragmas[key.toLowerCase()] = value
-                        /* groovylint-disable-next-line CatchArrayIndexOutOfBoundsException */
-                        } catch (ArrayIndexOutOfBoundsException ignored) {
-                            // ignore and move on to the next line
+        stage('Prepare Environment Variables') {
+            // TODO: Could/should these be moved to the environment block?
+            parallel {
+                stage('Get Commit Message') {
+                    steps {
+                        pragmasToEnv()
+                    }
+                }
```

Comment:
Mostly a reminder for myself that when I get the auto-test-tag magic working, we might need to rethink this a bit - or I rethink my approach

Reply:
I don't think this should have any impact.  This is just an optimization to store the commit pragmas into an env. variable rather than everywhere wanting to examine them running a `git` command.  Ultimately `cachedCommitPragma()` reads them from the env. var. if it exists and falls back to a `git` command if not.

And this use of `pragmasToEnv()` isn't really a change in behaviour.  It's just a more code efficient way of what was being done before.

Reply:
In my current version, I replace
```
sh(script: 'git show -s --format=%B',
                                            returnStdout: true).trim()
```
With a call to `get_commit_message.py`.

So now the question is how to call `get_commit_message.py` before `pragmasToEnv()`. But we can look at that later. The work is still some ways off